.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: java
+jdk:
+- oraclejdk8
+env:
+  global:
+  - secure: PLACEHOLDER=
+  - secure: PLACEHOLDER=
```

Comment:
After reading the travis blog and adding your username/password into this file these two placeholder lines can be deleted